DEVELOPMENT.md

```diff
@@ -16,7 +16,7 @@ This project uses `pip-tools`. To update requirements, do `pip install pip-tools
 
 Included `.code-workspace` has necessary options set (linting, formatting, tests, extensions) set for VS Code.
 
-Verify code style with `pre-commit run --all-files`, or use `pre-commit install` to generate an actual git hook.
+Verify code style with `pre-commit run --all-files`, or use `pre-commit install` and `pre-commit install --hook-type commit-msg` to generate an actual git hook.
```

Comment:
generate an actual git hook -> generate actual git hooks

But maybe it would be clearer to have own section for commit message style as it is technically not the same as code style?

Reply:
True. I can add a separate section for this (as in quality-result-gui).